apps/api/v2/src/modules/webhooks/guards/is-oauth-client-webhook-guard.ts

```diff
@@ -39,14 +39,10 @@ export class IsOAuthClientWebhookGuard implements CanActivate {
       throw new BadRequestException("webhookId parameter not specified in the request");
     }
 
-    if (!webhookId) {
+    if (!oAuthClientId) {
       throw new BadRequestException("oAuthClientId parameter not specified in the request");
     }
 
-    if (!user || !webhookId || !oAuthClientId) {
```

Comment:
Why are we getting rid of the user and webhookId checks?

Reply:
This was my my first thought as well, but if you check from line 34 to line 40 we already have checks for user and webhookId already present, and with the oAuthClientId also added I doubt if we'll ever reach this condition -`!user || !webhookId || !oAuthClientId`

Reply:
Exactly, hose lines are useless/unreachable since there is a check for each param

Reply:
Ah cool hadn't seen that piece. Nice.